### ISSUE TITLE
Translate Uyuni to your language

### ISSUE THREAD

**Issue by paususe** [uyuni-project/uyuni#2560]
This is a placeholder issue for HacktoberFest 2020

Check this wiki page and start translating!
https://github.com/uyuni-project/uyuni/wiki/Translating-Uyuni-to-your-language

No coding skills required.

Small components: do your component, get your shirt + sticker
Large components (e. g. Java): contact us (e. g. commenting here or in Gitter) and we'll agree on something.

**Comment by 0x5ubt13** [comment]
Hi, I'd like to collaborate translating to Spanish

**Comment by paususe** [comment]
@0x5ubt13 
Go for it! https://l10n.opensuse.org/projects/uyuni/

BTW, we are going to create a new uyuni-i18n mailing list for translators, it should be available in the coming days.

**Comment by bittin** [comment]
Started a bit with suggestions to Swedish as i heard about this on the OpenSUSE Online Conference, having problem with my Weblate account as the SUSE accounts migrated however

**Comment by paususe** [comment]
> Started a bit with suggestions to Swedish as i heard about this on the OpenSUSE Online Conference, having problem with my Weblate account as the SUSE accounts migrated however

Was your user account migrated to the new IDP?
https://news.opensuse.org/2020/05/09/community-account-migration/

If not, try requesting a password reset here:
https://idp-portal.suse.com/univention/self-service/#page=passwordreset

If you do not remember the e-mail account associated to your user account, you will need to create a new account here:
https://idp-portal.suse.com/univention/self-service/#page=createaccount

**Comment by bittin** [comment]
yep it was, works for logging in to bugzilla and other OpenSUSE sites, but when trying to log in to the weblate i get a breach warning as it thinks i use a new account or something

**Comment by bittin** [comment]
took care of some more swedish during susecon but far from done
